src/adera/Machines/Rocket.cpp

```diff
@@ -163,9 +170,86 @@ void SysMachineRocket::update_physics()
     }
 }
 
+void SysMachineRocket::attach_plume_effect(ActiveEnt ent)
+{
+    using Magnum::GL::Mesh;
+    using Magnum::GL::Texture2D;
+    using Magnum::Trade::ImageData2D;
+    using namespace Magnum::Math::Literals;
+
+    ActiveEnt plumeNode = entt::null;
+
+    auto findPlumeHandle = [&](ActiveEnt ent)
+    {
+        ACompHierarchy& node = m_scene.reg_get<ACompHierarchy>(ent);
+        if (node.m_name.compare(0, 9, "fx_plume_") == 0)
```

Comment:
```
static constexpr std::string_view comp = "fx_plume_";
if(node.m_name.compare(0, comp.size, comp.data()) == 0)
{ }
```
Or, my preference, which is yoda-conditionals

```
static constexpr std::string_view comp = "fx_plume_";
if(0 == node.m_name.compare(0, comp.size, comp.data()))
{ }
```

Reply:
Why do you prefer the latter? It looks weird. It's like saying
```cpp
3 = int myInt;
```
I know `==` is a commutative operation, but it still makes more sense to me to put the subject on the left.

Reply:
This is new to me, I like it. It makes it very obvious it's a comparison and not an assignment.

Reply:
because
```
if(ImARuntimeError = 4){}
if(4 = ImACompilerError){}
```

And yes, this is a real bug that I fixed in 25 year old code 10 years ago.

Reply:
What is it with you guys and horrendously ugly yet undeniably practical stylistic habits...

Reply:
When you spend several years working with octogenarians who wrote a big software product using K&R C on actual Unix in the 70s, and then never learned anything since that point in their lives, and make terrible programming errors constantly that just accidentally work because C89 (Yea, they upgraded... once) is trash as possible...

you learn how to defend yourself from your co-workers.